stdlib/src/array.rs

```diff
@@ -1,4 +1,31 @@
-pub(crate) use array::make_module;
+use rustpython_vm::{PyObjectRef, VirtualMachine};
+
+pub(crate) fn make_module(vm: &VirtualMachine) -> PyObjectRef {
+    let module = array::make_module(vm);
+
+    let array = module
+        .get_attr("array", vm)
+        .expect("Expect array has arrat type.");
+
+    // TODO: RUSTPYTHON
+    // FIXME: it should import 'collections.abc' instead of '_collection_abc'
+    let collections_abc = vm
+        .import("_collections_abc", None, 0)
+        .expect("Expect collections.abc exist.");
+    let mutable_sequence = collections_abc
+        .get_attr("MutableSequence", vm)
+        .expect("Expect collections.abc has MutableSequence type.");
```

Comment:
the equivalent for us would probably be `getattr` on `collections_abc` to get the `abc` submodule, our `import` doesn't exactly work like `PyImport_ImportModule` (which, as [its docs](https://docs.python.org/3/c-api/import.html#c.PyImport_ImportModule) state, returns the named module after the `.`), so, something like:

```suggestion
    let collections_abc = vm
        .import("collections.abc", None, 0)
        .expect("Expect collections exist.");
    let abc = collections_abc.get_attr("abc", vm).expect("Expect collections.abc has abc submodule.");
    let mutable_sequence = abc
        .get_attr("MutableSequence", vm)
        .expect("Expect collections.abc has MutableSequence type.");
```

would probably be closest to `PyImport_ImportModule("collections.abc")`.